src/mediumComplexityProblem1/IncomeTaxCalculation.java

```diff
@@ -0,0 +1,42 @@
+package mediumComplexityProblem1;
+
+import java.util.Scanner;
+
+public class IncomeTaxCalculation {
+    private double tax;
+    private double it;
+
+    public void findIncomeTax() {
```

Comment:
take the salary as a Argument. In main method add scanner method.

Reply:
Added taxable income as argument from main method